### ISSUE TITLE
HueBot not available in group calls

### ISSUE THREAD

**Issue by PavelGladko** [Microsoft/BotBuilder-RealTimeMediaCalling#22]
I have created a bot on basis of Hue bot example, and till last week it was available during group calls. Starting from this week bot is not available during group calls. 

Could someone please confirm that access to the group calls was closed in passed 10 days?
If so, any ideas when bots would be available for group cals again? 

Thanks in advance!

**Comment by gfclaveria** [comment]
@PavelGladko I have the same issue with MeetingScreenshotBot just right now when i tried to deploy the bot i have with my modifications. :-(


_To Concern Developers (Skype Real-Time Media team):_
Is this the time for Us to stop working/tinkering these bot samples that concerns groups calls?
Was the reported (but not officially announced)   **Disabling of bots for Group Calls** already implemented?

**Comment by frleger** [comment]
@PavelGladko @gfclaveria 

Group calling for media bots has been disabled because the Skype clients do not have currently have the appropriate privacy controls in place to inform the participants that a bot can hear and/or see them. There is no timeframe available regarding when calling/real-time media bots will be supported in group calls. For now, calling and real-time media bots are allowed only in 1-to-1 Skype calls.

**Comment by gfclaveria** [comment]
@frleger 

Hi, another question.  I know that real-time media doesn't support Skype For Business as what I have been told.  I just want to know: 

Is the **Trusted Application API/SDK** has this real-time media capabilities?  Features seem promising.

**Comment by ssulzer** [comment]
@gfclaveria 

No, real-time media calling capabilities are not available with the Trusted Application API.  The Trusted Application SDK is in Developer Preview supporting Guest Meeting Join scenarios only.  Supporting real-time media calling for Enterprise/Business bots is under consideration for future versions. Thanks.

**Comment by gfclaveria** [comment]
@ssulzer Thanks!
